package.json

```diff
@@ -49,12 +49,12 @@
     "src"
   ],
   "dependencies": {
-    "immer": "^1.10.5",
+    "immer": "^2.1.5",
```

Comment:
This one may be no? Immer now resolves Promises returned from produce.

Reply:
Yeah, let's stay on 1.x for the time being.  I know they've been making some changes over there, and I'm not clear what all is different.  (For example, there was discussion of ignoring values returned from `produce()` in favor of just draft mutations, but I don't know how that turned out.)

Reply:
Cool cool, also saw they're like in the middle of the WIP 3.x release over there too 😆

Reply:
I will update to latest 1.x

Reply:
So having said that, someone else just filed an "Update to Immer 2.x" PR and I merged it, so let's go with that.